features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1170,7 +1170,10 @@
     "BasicAuthRequestPathAuthenticator",
     "OAuthRequestPathAuthenticator"
   ],
-  "console.ui.hiddenConnectionTemplates": [ "swe-idp" ],
+  "console.ui.hiddenConnectionTemplates": [
```

Comment:
This will only hide the template from Identity Server, if you need to hide it from cloud product as well, we need to put an another PR.